fedimint-api-client/src/api.rs

```diff
@@ -1278,9 +1344,10 @@ where
     }
 
     /// Creates a new API client
-    pub fn new_with_client(peers: Vec<(PeerId, SafeUrl)>) -> Self {
+    pub fn new_with_client(peers: Vec<(PeerId, SafeUrl)>, self_peer_id: Option<PeerId>) -> Self {
```

Comment:
What does the `with_client` stand for here? That we already initialize the inner clients?